main.go

```diff
@@ -191,16 +228,30 @@ func main() {
 		VerboseLog:              cfg.VerboseLog,
 	})
 	if err != nil {
-		failf(fmt.Sprintf("Automatic code signing failed: %s", err))
-	}
+		if !fallbackProfileDownloader.IsAvailable() {
+			failf(fmt.Sprintf("Automatic code signing failed: %s", err))
+		}
 
-	if err := project.ForceCodesignAssets(distribution, codesignAssetsByDistributionType); err != nil {
-		failf(fmt.Sprintf("Failed to force codesign settings: %s", err))
+		logger.Println()
+		logger.Warnf("Automatic code signing failed: %s", err)
+		logger.Println()
+		logger.Infof("Falling back to manually managed codesigning assets.")
+		if err := prepareManualAssets(logger, certs, fallbackProfileDownloader, assetWriter); err != nil {
+			failf(fmt.Sprintf("Automatic code signing failed: %s", err))
+		}
+	} else {
+		if err := project.ForceCodesignAssets(distribution, codesignAssetsByDistributionType); err != nil {
+			failf(fmt.Sprintf("Failed to force codesign settings: %s", err))
+		}
 	}
 
 	// Export output
 	fmt.Println()
 	logger.Infof("Exporting outputs")
+	if len(codesignAssetsByDistributionType) == 0 {
+		logger.Warnf("Skipping output export, as fallback provisioning profiles are used.")
+		return
```

Comment:
Why do we skipp the output export, instead of exporting data about the fallback signing files?